README.md

```diff
@@ -565,4 +601,4 @@ Calculate a translate matrix
 - [rodrigoapereira](https://github.com/rodrigoapereira)
 - [formatlos](https://github.com/formatlos)
 - [benhjames](https://github.com/benhjames)
-
+- [hillin](https://github.com/hillin)
```

Comment:
Can you add this line to README.template.hbs? Otherwise this row will be lost during the next build, since the README file is autogenerated.

Reply:
Sure.